.github/workflows/status.yaml

```diff
@@ -85,13 +85,35 @@ jobs:
                 state: CUSTOM_STATE = 'success'
             } = contentJSON;
 
+            async function fetchAllJobs(owner, repo, run_id) {
```

Comment:
Use `github.paginate` instead.

There's an example here: https://github.com/actions/github-script?tab=readme-ov-file#welcome-a-first-time-contributor

Reply:
Wow, didn't know about that. That simplifies a lot of the code. Thanks @raydouglass !